snuba/datasets/configuration/discover/entities/discover_transactions.yaml

```diff
@@ -244,7 +244,6 @@ storages:
               - exception_frames.colno
               - exception_frames.lineno
               - exception_frames.stack_level
-              - exception_main_thread
```

Comment:
@barkbarkimashark I noticed you removed this only from `discover.yaml`, should I remove it from here as well?

Reply:
I guess this should stay, `exception_main_thread` should always be nullable for transactions